### ISSUE TITLE
Unable to make it work property

### ISSUE THREAD

**Issue by fstarred** [jeff-1amstudios/c64-smooth-scrolling#1]
Guess it's an acme source because I haven't found any information about the cross assembler used, however the program crashes with a syntax error.
If I try to start it manually by typing "sys 49152" the program behaves in an odd way.

**Comment by w-A-L-L-e** [comment]
Yep same here, tried this last night. It first crashes then a list + sys49152 does start it but garbage lines appear at top and it's not smooth scrolling...
<img width="676" alt="Screenshot 2019-03-26 at 15 02 05" src="https://user-images.githubusercontent.com/710803/55003458-c2214000-4fd8-11e9-9f41-c6ec08ab98e3.png">
<img width="663" alt="Screenshot 2019-03-26 at 15 02 21" src="https://user-images.githubusercontent.com/710803/55003459-c2b9d680-4fd8-11e9-979a-eda6dd915945.png">
<img width="658" alt="Screenshot 2019-03-26 at 15 02 28" src="https://user-images.githubusercontent.com/710803/55003460-c2b9d680-4fd8-11e9-913d-7ae688bab95c.png">
<img width="660" alt="Screenshot 2019-03-26 at 15 02 36" src="https://user-images.githubusercontent.com/710803/55003461-c2b9d680-4fd8-11e9-9284-ba48e8dddea1.png">

**Comment by climbus** [comment]
I have fixed this bug. There was a wrong subroutine name. I created a pull request with fix.

https://github.com/jeff-1amstudios/c64-smooth-scrolling/pull/4

I have a working version in kick assembler, and I'll do my best to publish it.
